src/client/editor/components/StandardMaterialComponent.js

```diff
@@ -62,6 +62,8 @@ export default class StandardMaterialComponent extends SaveableComponent {
       this._object[map] = texture;
     } catch (e) {
       // TODO Should show warning on texture property when this happens.
+      const resource = this._mapTypesToTextureType(map);
+      this.updateResourceValidation(resource, false);
```

Comment:
Actually, this was incorrect to begin with. This `try-catch` doesn't do anything. Validation handling should be in the `onError` callback above.